CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.5.1)
```

Comment:
3.5.1 is default for Ubuntu 16.
If you want to, we can decrease this version requirement, but I'm not sure anyone needs that.